examples/helm/README.md

```diff
@@ -0,0 +1,65 @@
+# crunchycrdcluster
+
+This is a working example of how to create a cluster via the crd workflow
+using a helm chart
+
+## Assumptions
+This example assumes you have the Crunchy PostgreSQL Operator installed
+in namespace called pgo.  
+
+## Helm
+Helm will also need to be installed for this example to run
+
+## Documenation
+Please see the documentation for more guidance using custom resources:
+
+https://access.crunchydata.com/documentation/postgres-operator/latest/custom-resources/
+
+
+## Example set up and execution
+create a cert directy and generate certs
+```
+cd postgres-operator/examples/helm/crunchycrdcluster
+
+mkdir cert
+
+# this variable is the name of the cluster being created
+export pgo_cluster_name=hippo
+
+# generate a SSH public/private keypair for use by pgBackRest
+ssh-keygen -t ed25519 -N '' -f "${pgo_cluster_name}-key"
+
+```
+The cluster can bedeployed in the pgo namespace but for this example
+we are going to create a Crunchy PostgreSql cluster in a namespace 
+called test1
+```
+pgo create namespace test1
```

Comment:
Let's eliminate the create namespace step. Allow for the cluster to be created within the same namespace as the Operator.

Reply:
updated the namespace to pgo